kubernetes/create-config-maps.sh.j2

```diff
@@ -20,6 +20,14 @@ kubectl create secret generic concent-api-secrets            \
     --from-file=secrets.py=concent-secrets/secrets.py        \
     --from-literal=__init__.py=
 
+{% if gke.cluster == "concent-dev" or gke.cluster == "concent-staging" %}
+kubectl create secret generic signing-service-secrets                      \
+    --from-file=ethereum-secrets.py=concent-secrets/ethereum-secrets.py    \
+    --from-literal=__init__.py=
```

Comment:
Now that https://github.com/golemfactory/concent/issues/671 is done, please switch to passing the secret key in an environment variable.

Reply:
Done.